app/src/main/res/layout/screen_manual_patient_entry.xml

```diff
@@ -133,8 +133,8 @@
           android:padding="@dimen/spacing_4"
           android:textAppearance="?attr/textAppearanceBody1Numeric"
           android:textColor="?attr/colorOnBackground"
-          tools:drawableStart="@drawable/patient_id_card"
-          tools:drawableTint="@color/color_on_surface_67"
+          app:drawableStartCompat="@drawable/patient_id_card"
+          app:drawableTint="@color/color_on_surface_67"
```

Comment:
We still use a `TextView` here, so how come this attribute is available now? 🤔

Reply:
Material inflater will automatically converts this to `MaterialTextView` which extends `AppCompatTextView`. So it's safe to use this. AppCompat should have been doing the same thing as well, but I am not sure what the problem was earlier 🤷🏾